lib/workload/stateless/postgres_manager/construct/postgresManager.ts

```diff
@@ -0,0 +1,117 @@
+import { Duration } from 'aws-cdk-lib';
+import { Construct } from 'constructs';
+import * as lambda from 'aws-cdk-lib/aws-lambda';
+import * as nodejs from 'aws-cdk-lib/aws-lambda-nodejs';
+import * as secretsmanager from 'aws-cdk-lib/aws-secretsmanager';
+import * as ec2 from 'aws-cdk-lib/aws-ec2';
+import * as iam from 'aws-cdk-lib/aws-iam';
+import * as rds from 'aws-cdk-lib/aws-rds';
+import * as ssm from 'aws-cdk-lib/aws-ssm';
+import { DbAuthType, MicroserviceConfig } from '../function/utils';
+
+export type PostgresManagerConfig = {
+  masterSecretName: string;
+  dbClusterIdentifier: string;
+  microserviceDbConfig: MicroserviceConfig;
+  clusterResourceIdParameterName: string;
+};
+
+export type PostgresManagerStackProps = PostgresManagerConfig & {
+  vpc: ec2.IVpc;
+  lambdaSecurityGroup: ec2.ISecurityGroup;
+};
+
+export class PostgresManager extends Construct {
+  constructor(scope: Construct, id: string, props: PostgresManagerStackProps) {
+    super(scope, id);
+
+    const { dbClusterIdentifier, microserviceDbConfig } = props;
+
+    const masterSecret = secretsmanager.Secret.fromSecretNameV2(
+      this,
+      'RdsMasterSecret',
+      props.masterSecretName
+    );
+
+    const dbClusterResourceId = ssm.StringParameter.valueFromLookup(
+      this,
+      '/orcabus/db-cluster-resource-id'
```

Comment:
Could reuse the constant `dbClusterResourceIdParameterName` from `constants.ts` here?

Reply:
Yes, thanks! miss this one out